601-spring-data-primitive-types/src/main/java/io/quarkus/qe/spring/data/CatResource.java

```diff
@@ -1,9 +1,10 @@
 package io.quarkus.qe.spring.data;
 
-import javax.ws.rs.GET;
-import javax.ws.rs.Path;
-import javax.ws.rs.PathParam;
-import javax.ws.rs.Produces;
+import io.quarkus.qe.spring.data.model.Cat;
+import java.util.List;
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.*;
```

Comment:
Please use individual imports, IDE can be hinted to avoid star imports by increasing the default (usually 5) number of imports before doing .*